### ISSUE TITLE
[A11y] Pagination Bullets get's invisible/hidden.

### ISSUE THREAD

**Issue by ravigajjala** [nolimits4web/Swiper#8089]
### Check that this is really a bug

- [x] I confirm

### Reproduction link

https://github.com/user-attachments/assets/1d7cb387-29a8-46fe-8dd1-82bfba0f400a

### Bug description

1. I encountered an issue were pagination bullets(dot's) get's hidden/invisible when we try to slide using Voice Over on IOS Devices & Talk Back on Android Devices.
2. If Voice over/talk back is disabled in-between slides then i can no longer manually swipe back to previous slide & have to finish swipe until end of all available slides.However on the last slide swiper becomes  Un-responsiveness.

https://github.com/user-attachments/assets/1d7cb387-29a8-46fe-8dd1-82bfba0f400a

### Expected Behavior

Pagination bullets should be always visible though we are sliding through voiceover/talk back.Basically experience should be same as the user swipes manually with their finger.

### Actual Behavior

Pagination bullets get disappear & swiper becomes unresponsiveness on the last available slide.

### Swiper version

12.0.1

### Platform/Target and Browser Versions

All Browser's.

### Validations

- [x] Follow our [Code of Conduct](https://github.com/nolimits4web/swiper/blob/master/CODE_OF_CONDUCT.md)
- [x] Read the [docs](https://swiperjs.com/swiper-api).
- [x] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.
- [x] Make sure this is a Swiper issue and not a framework-specific issue

### Would you like to open a PR for this bug?

- [ ] I'm willing to open a PR
